components/Heading.tsx

```diff
@@ -0,0 +1,15 @@
+import { Typography } from "@mui/material";
+import {theme} from "../theme";
+
+interface HeadingProps {
+  component: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+  content: string;
+}
```

Comment:
It can increase code readability if u will extract interfaces to their own folder